slick/src/main/scala/slick/jdbc/OracleProfile.scala

```diff
@@ -61,6 +67,8 @@ trait OracleProfile extends JdbcProfile {
 
   override protected def computeCapabilities: Set[Capability] = (super.computeCapabilities
     - RelationalCapabilities.foreignKeyActions
+    - RelationalCapabilities.createIfNotExists
+    - RelationalCapabilities.dropIfExists
```

Comment:
I think client side implementation is fine. The flags just identify if Slick is able to do this using whatever method. So I don't think we need to remove them. In fact it sounds like we can get rid of them. Back then I only suggested them as an alternative in case it was impossible to do with one of the drivers.